spec/system/event_log_spec.rb

```diff
@@ -86,14 +49,8 @@
     end
 
     context 'when all questions are answered incorrectly' do
-      before do
-        visit '/modules/alpha/content-pages/what-to-expect'
-
-        # OPTIMIZE: the setup for this context leverages the AST schema which supports future changes to content
-        ContentTestSchema.new(mod: alpha).call(pass: false).compact.each do |content|
-          content[:inputs].each { |args| send(*args) }
-        end
-      end
+      # include_context 'with alpha unhappy path'
```

Comment:
Is this a comment that needs removing?

Reply:
It's out of date, the comment was to offer you the YAML based alternative